images/cross/Makefile

```diff
@@ -20,6 +20,8 @@ CACHE_IMAGES = $(IMAGES)
 
 BASEIMAGE = $(BUILD_REGISTRY)/base-$(GOARCH)
 
+makeargs=-j$(getconf _NPROCESSORS_ONLN)
```

Comment:
@bassam better I guess :)

Reply:
where is `makeargs` used? Did you mean `MAKEFLAGS`?

Reply:
why getconf instead of nproc?  nproc seems to take into account CPU restrictions, which I'm not sure _NPROCESSORS_ONLN does; see https://www.gnu.org/software/coreutils/manual/html_node/nproc-invocation.html

Reply:
@dmick used `_NPROCESSORS_ONLN` just in case `coreutils` is not installed.
@bassam I might have made a mistake, let me check.